node/beacon/src/authority.rs

```diff
@@ -440,4 +506,26 @@ mod test {
         assert_eq!(find_threshold(&[1000, 1, 1, 1, 1, 1, 1, 1, 1, 1], 1).unwrap(), 1000);
         assert!(find_threshold(&[1, 1, 2], 100).is_err());
     }
+
+    #[test]
+    fn test_write_to_storage() {
```

Comment:
this test doesn't acutally check that you restored the state correctly. please test here:
  - create blockchain with storage and write blocks
  - re-create blockchain with the same storage and check that `get_authorities` is the same.

Reply:
Fixed